sockeye/training.py

```diff
@@ -557,7 +569,8 @@ def _adjust_learning_rate(self, has_improved: bool):
                     self.sockeye_model.load_parameters(filename=self.best_params_fname, device=self.device)
                 if os.path.exists(self.best_optimizer_state_fname):
                     self._load_optimizer_state(self.best_optimizer_state_fname)
-            lr = scheduler.lr
+            # Assume same learning rate for all param groups
```

Comment:
Do you document somewhere (maybe in lr_scheduler.py) what parameter groups are?

Reply:
Added a note to the docstring for `LearningRateScheduler` on where to find information about PyTorch's optimizer/scheduler APIs.